compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skiko.kt

```diff
@@ -126,7 +124,7 @@ fun defaultTestDispatcher() = UnconfinedTestDispatcher()
  */
 @ExperimentalTestApi
 @OptIn(InternalTestApi::class, InternalComposeUiApi::class)
-class SkikoComposeUiTest @InternalTestApi constructor(
+open class SkikoComposeUiTest @InternalTestApi constructor(
```

Comment:
Why I think it's okay to make it `open`:

- it's experimental, so such changes are okay. If somone uses/extends it, it's at their own risk.
- SkikoComposeUiTest is not available in common tests (with Android), which I believe is the main place where the tests should be added in Compose Multiplatform case. Those tests will use only `ComposeUiTest` interface.
- In the future, I believe we will need to update this API, since the intention is to not expose "skiko". SkikoComposeUiTest exposes `scene`, which is convenient, but it's not supposed to be used outside of Compose - InternalComposeUiApi.